osquery/CMakeLists.txt

```diff
@@ -33,6 +33,12 @@ else()
     -Wno-unused-parameter
     -Wno-gnu-case-range
   )
+  if (CMAKE_CXX_COMPILER MATCHES "clang")
```

Comment:
Ah, I meant the whole body, the CMakeLists.txt in the top level should have added this compile option right?